renderdoc/driver/gl/gl_renderstate.cpp

```diff
@@ -606,7 +606,7 @@ void GLRenderState::MarkDirty(WrappedOpenGL *driver) const
         if(type == eGL_RENDERBUFFER)
           manager->MarkDirtyResource(RenderbufferRes(ctx, name));
         else
-          manager->MarkDirtyResource(TextureRes(ctx, name));
+          manager->MarkDirtyWithWriteReference(TextureRes(ctx, name));
```

Comment:
I can't comment on the actual line - but in this function there are write references to buffers which aren't being propagated. E.g. the SSBO bindings are calling `MarkDirtyResource` instead of `MarkDirtyWithWriteReference` which means if an SSBO is bound then written every frame but the binding isn't touched, it will be considered persistent incorrectly.